charts/service-binding-operator/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: service-binding-operator
+description: A Helm chart to deploy service binding operator
+type: application
+version: 1.0.0
+appVersion: "1.0.1"
```

Comment:
similar to OLM bundle, the chart needs to specify what is minimal supported kube version, add same set of keywords and other metadata available in OLM bundle.

/hold

Reply:
kubeVersion: '>= 1.21.0' for OCP 4.8 right? https://access.redhat.com/solutions/4870701

Reply:
CI tests with 1.19: https://github.com/redhat-developer/service-binding-operator/blob/master/.github/workflows/pr-checks.yaml#L13 so why putting the bar higher here? Also, there are plenty of useful metadata https://github.com/redhat-developer/service-binding-operator/blob/master/config/manifests/bases/service-binding-operator.clusterserviceversion.yaml that could be reused here as well, at least this is what I would expect from a chart carrying 1.0 version.

Reply:
Majority of data looks like field used by OLM ,as the target for our chart is non OLM i think it is fine just to mention the kubeversion.